ecommerce_worker/fulfillment/v1/tasks.py

```diff
@@ -29,33 +31,46 @@ def _retry_order(self, exception, max_fulfillment_retries, order_number):
 
 @shared_task(bind=True, ignore_result=True)
 def fulfill_order(self, order_number, site_code=None, email_opt_in=False):
-    """Fulfills an order.
+    """
+    Fulfills an order.
 
     Arguments:
         order_number (str): Order number indicating which order to fulfill.
 
     Returns:
         None
     """
+
+    api_url = urljoin(
+        get_configuration('ECOMMERCE_API_ROOT', site_code=site_code),
```

Comment:
trailing slash to the base_url param is recommended to be added to prevent creating an incorrect url.